README.md

```diff
@@ -1,147 +1,78 @@
-# proxy-client-svelte
-
-PoC for a Svelte SDK for [Unleash](https://www.getunleash.io/) based on the official [proxy-client-react](https://github.com/Unleash/proxy-client-react).
-
-# DISCLAIMER:
-
-This library is meant to be used with the [unleash-proxy](https://github.com/Unleash/unleash-proxy). The proxy application layer will sit between your unleash instance and your client applications, and provides performance and security benefits. DO NOT TRY to connect this library directly to the unleash instance, as the datasets follow different formats because the proxy only returns evaluated toggle information.
-
 # Installation
 
 ```bash
 npm install @unleash/proxy-client-svelte
-// or
+# or
 yarn add @unleash/proxy-client-svelte
 ```
 
-# Initialization
-
-Import the provider like this in your entrypoint file (typically index.svelte):
-
-```jsx
-<script lang="ts">
-	let FlagProvider;
-
-	onMount(async () => {
-		const proxyClientSvelte = await import('@unleash/proxy-client-svelte');
-		({ FlagProvider } = proxyClientSvelte);
-	});
+# How to use
 
-	const config = {
-		url: 'https://UNLEASH-INSTANCE/api/frontend',
-		clientKey: 'CLIENT—SIDE—API—TOKEN',
-		refreshInterval: 15,
-		appName: 'your-app-name',
-	};
-</script>
+## Initialize the client
 
-<svelte:component this={FlagProvider} {config}>
-	<App />
-</svelte:component>
-```
+Prepare [Unleash Proxy](https://docs.getunleash.io/reference/unleash-proxy) secret
+or [Frontend API Access](https://docs.getunleash.io/reference/front-end-api) token.
 
-Alternatively, you can pass your own client in to the FlagProvider:
+Import the provider like this in your entrypoint file (typically index.svelte):
 
-```jsx
+```svelte
 <script lang="ts">
-	import { UnleashClient } from '@unleash/proxy-client-svelte';
-
-	let FlagProvider;
-
-	onMount(async () => {
-		const proxyClientSvelte = await import('@unleash/proxy-client-svelte');
-		({ FlagProvider } = proxyClientSvelte);
-	});
+	import { FlagProvider } from '@unleash/proxy-client-svelte';
 
 	const config = {
-		url: 'https://UNLEASH-INSTANCE/api/frontend',
-		clientKey: 'CLIENT—SIDE—API—TOKEN',
-		refreshInterval: 15,
-		appName: 'your-app-name',
+		url: '<unleash-url>/api/frontend', // Your front-end API URL or the Unleash proxy's URL (https://<proxy-url>/proxy)
```

Comment:
OR edge

Reply:
Addressed in https://github.com/Unleash/proxy-client-svelte/pull/11/commits/c1bf68ec276f666e3a3c7bc3aaf89905a9d05bad